azure-iot-device/tests/iothub/auth/conftest.py

```diff
@@ -0,0 +1,9 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from .shared_auth_fixtures import *
+
+# from .shared_auth_tests import *
```

Comment:
>shared_auth_tests [](start = 8, length = 17)

Do we need this ?

Reply:
Nope

---
In reply to: [294488050](https://github.com/Azure/azure-iot-sdk-python-preview/pull/110#discussion_r294488050) [](ancestors = 294488050)